### PR TITLE
url change to form

### PR DESCRIPTION
**Pull request by sakulachi8**

### CODE REVIEW

File: IsHumanFunction/isHuman.cs [hunk 1]
```diff
@@ -15,9 +15,9 @@ namespace Company.Function
 {
     public static class isHuman
     {
-        [FunctionName("isHuman")]
+        [FunctionName("Form")]
         public static async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "isHuman")] HttpRequest req,
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "form")] HttpRequest req,
```
Comment (misterhackney) [inline IsHumanFunction/isHuman.cs:20]: your capitalization is different for line 18 and line 20 for the word "form"
Please confirm this is done on purpose before I confirm the PR.

Comment (sakulachi8) [reply IsHumanFunction/isHuman.cs:20]: It doesnt matter, please proceed,

Comment (sakulachi8) [reply IsHumanFunction/isHuman.cs:20]: routing is not case sensitive, even function name is not case sensitive,

Comment (sakulachi8) [reply IsHumanFunction/isHuman.cs:20]: If you ask me to change, i will.

Comment (misterhackney) [reply IsHumanFunction/isHuman.cs:20]: please change.  Always act case sensitive, even when software doesn't.
Someone might port this to another language that is case-sensitive. We do not want to cause more problems for them in trying to interpret our code.

Comment (sakulachi8) [reply IsHumanFunction/isHuman.cs:20]: Ok changing,

**Pull request merged**